.cirrus.yml

```diff
@@ -19,6 +19,9 @@ release_ready_task:
     - ./configure $DEFAULT_CONFIG_OPTS
   checks_script:
     - make -j$(nproc) -s distcheck DISTCHECK_CONFIGURE_FLAGS="${DEFAULT_CONFIG_OPTS}"
+  always:
+    make_check_artifacts:
+      path: "**/*.log"
```

Comment:
The way that CirrusCI works is that is looks for specific suffixes. Above it's `_script` for build steps, for uploading artifacts it's `_artifacts`. The first part can be chosen by us. This will show up in the UI as "Upload make_check artifacts" which is IMHO quite clear.

If you prefer, I'm happy to rename `checks_script` with `make_distcheck_script` for consistency.

Reply:
If it's indeed for `make distcheck`, then it would be good to rename it before merging.   Approved either way.

Reply:
Done.